src/cli/backup/sharepoint.go

```diff
@@ -181,22 +183,57 @@ func createSharePointCmd(cmd *cobra.Command, args []string) error {
 
 	sel := sharePointBackupCreateSelectors(site)
 
-	bo, err := r.NewBackup(ctx, sel)
+	sites, err := m365.UserIDs(ctx, acct)
 	if err != nil {
-		return Only(ctx, errors.Wrap(err, "Failed to initialize SharePoint backup"))
+		return Only(ctx, errors.Wrap(err, "Failed to retrieve M365 users"))
```

Comment:
Sharepoint should retrieve sites, not userIDs.

Reply:
huh and I even went through the trouble to make a function to get the sites and all